gaia/core.py

```diff
@@ -66,4 +69,18 @@ def get_config(config_file=None):
 
 config = get_config()
 
-sqlengines = {}
+
+def get_plugins():
+    """
+    Load and return a list of installed plugin modules
+    :return: list of plugin modules
+    """
+    installed_plugins = []
+    for ep in pr.iter_entry_points(group='gaia.plugins'):
+        module = ep.load()
+        importlib.import_module(module.__name__)
+        installed_plugins.append(module)
+        plugin_config = get_config(
+            config_file='{}/gaia.cfg'.format(os.path.dirname(module.__file__)))
```

Comment:
@mbertrand Can we do this through a known function defined inside the module? e.g.
```python
if hasattr(module, 'get_config'): 
    config.update(module.get_config())
```
That way the core gaia code doesn't enforce a particular config mechanism on the plugins  (e.g. gaia.cfg in root directory).